src/payment/strategies/amazon-maxo/amazon-maxo.ts

```diff
@@ -0,0 +1,66 @@
+export type EnvironmentType = 'PRODUCTION' | 'TEST';
+
+export interface AmazonMaxoOptions {
+    environment: EnvironmentType;
+}
+
+export interface AmazonMaxoSDK {
+    Pay: AmazonMaxoClient;
+}
+
+export interface AmazonMaxoClient {
+    renderButton(containerId: string, params: AmazonMaxoButtonParams): HTMLElement;
+    bindChangeAction(): null;
```

Comment:
This return value is null? or void?

Reply:
Fixed, the return type is void ♻️

Reply:
If the return type is void it can be like this
```suggestion
    bindChangeAction(): void;
```